_usage/command-line.md

```diff
@@ -195,9 +193,9 @@ works with Microsoft SQL Server as well as PostgreSQL.
 
 You can also use PostgreSQL's `psql` to connect to Babelfish. However, this is
 **strongly** discouraged, and the behavior will be unpredictable. It is
-therefore not recommended to to that &ndash; in fact it can be pretty dangerous.
+not recommended.
```

Comment:
Check with one of the core dev team, but I think this can be removed - I don't think there is a problem connecting to Babelfish with psql:

You can also use PostgreSQL's `psql` to connect to Babelfish. However, this is **strongly** discouraged, and the behavior will be unpredictable.  It is not recommended.